plugins/git/README.md

```diff
@@ -22,7 +22,7 @@ plugins=(... git)
 | gb                   | git branch                                                                                                                       |
 | gba                  | git branch -a                                                                                                                    |
 | gbd                  | git branch -d                                                                                                                    |
-| gbda                 | git branch --no-color --merged \| command grep -vE "^(\+\|\*\|\s*(master\|develop\|dev)\s*$)" \| command xargs -n 1 git branch -d |
+| gbda                 | git branch --no-color --merged \| command grep -vE "^(\+\|\*\|\s*(master\|development|develop\|dev)\s*$)" \| command xargs -n 1 git branch -d |
```

Comment:
Hi, you forgot a backslash after `development`

Reply:
Thx, this is now fixed